workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -0,0 +1,122 @@
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Improbable.Gdk.Tools
+{
+    public static class LocalLaunch
+    {
+        private static readonly string
+            SpatialProjectRootDir = Path.GetFullPath(Path.Combine(Application.dataPath, "..", "..", ".."));
+
+        // Windows: The exit code is 0xc000013a when the user closes the console window, or presses Ctrl+C.
+        private const int WindowsCtrlCExitCode = -1073741510;
+
+        // Unix-like: The exit code is 128 + SIGINT (2).
+        private const int UnixSigIntExitCode = 128 + 2;
+
+        [MenuItem("Improbable/Spatial/Build worker configs")]
+        private static void BuildConfigMenu()
+        {
+            Debug.Log("Building worker configs...");
+            EditorApplication.delayCall += BuildConfig;
+        }
+
+        [MenuItem("Improbable/Spatial/Local launch")]
+        private static void LaunchMenu()
+        {
+            Debug.Log("Launching SpatialOS locally...");
+            EditorApplication.delayCall += () =>
+            {
+                BuildConfig();
+                Launch();
+            };
+        }
+
+        public static void BuildConfig()
+        {
+            using (new ShowProgressBarScope("Building worker configs..."))
+            {
+                // Run from the root of the project to build all available worker configs.
+                RedirectedProcess.RunIn(SpatialProjectRootDir, Common.SpatialBinary, "build", "build-config",
+                    "--json_output");
+            }
+        }
+
+        public static void Launch()
+        {
+            BuildConfig();
+            var processInfo = new ProcessStartInfo("spatial", "local launch")
+            {
+                CreateNoWindow = false,
+                UseShellExecute = true,
+                WorkingDirectory = SpatialProjectRootDir
+            };
+
+            var process = Process.Start(processInfo);
+
+            if (process == null)
+            {
+                return;
+            }
```

Comment:
What's would cause process to be null immediately after start?

Reply:
Ah right - do we want to log that something went wrong?

Reply:
Yep, done!